### ISSUE TITLE
Resolve "WCAG mds-avatar and mds-img" - [merged]

### ISSUE THREAD

**Issue by nicolatamburini** [magma-design-system/magma#408]
***In GitLab by @vittorio.vittori on Oct 11, 2022, 16:43 GMT+2:***

_Merges 134-wcag-mds-avatar-and-mds-img -> dev_

This action adds the feature to the mds-img element to automatically add the alt attribute if this is not done manually.

The feature serves to mitigate the lack of the alt attribute from the img element contained within mds-img.

Closes #134

**Assignees:** vittorio.vittori

**Reviewers:** andrea.pruccoli

**Approved by:** andrea.pruccoli

*Migrated from GitLab: https://gitlab.com/maggiolispa/ricerca-sviluppo-new-media/magma/design-system/-/merge_requests/68*

**Comment by nicolatamburini** [comment]
***In GitLab by @vittorio.vittori on Oct 11, 2022, 16:43 GMT+2:***

requested review from @andrea.pruccoli

**Comment by nicolatamburini** [comment]
***In GitLab by @andrea.pruccoli on Oct 17, 2022, 14:55 GMT+2:***

approved this merge request

**Comment by nicolatamburini** [comment]
***In GitLab by @vittorio.vittori on Oct 17, 2022, 15:03 GMT+2:***

mentioned in commit 504fa3b10bdf089786a05dccc862ded8ee372c45
